Composer/packages/extensions/intellisense/src/components/CompletionList.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { CSSProperties } from 'react';
+import { CompletionItem } from 'vscode-languageserver-types';
+import CompletionElement from './CompletionElement';
+
+const styles: Record<string, CSSProperties> = {
+  completionList: {
+    position: 'absolute',
+    top: 32,
+    left: 0,
+    maxHeight: '300px',
+    width: '100%',
+    backgroundColor: 'white',
+    overflowY: 'auto',
+    overflowX: 'hidden',
+    boxShadow: '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)',
+    zIndex: 2000,
+  },
+};
+
+const CompletionList = React.forwardRef<
```

Comment:
It is worth a comment why you are exposing the completionList as a div element for manipulation outside the component.  Consider if you want to provide an explicit method on this rather than expose the root is a div.